webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -234,7 +236,7 @@ const NavigationSideBar: React.FC = () => {
             }
             offset={settings.navbarCollapsed ? { x: -8, y: 16 } : { x: 16, y: -8 }}
             placement={settings.navbarCollapsed ? Placement.RightTop : Placement.BottomLeft}>
-            <AvatarCard className={css.user} darkLight={ui.darkLight} user={authUser} />
+            <AvatarCard className={css.user} darkLight={ui.darkLight} user={currentUser} />
```

Comment:
Can we move the `Loadable.match` here to show empty string as a fallback, instead of `<AvatarCard user={undefined} />` . I expect `NotLoaded` to be rare because we set the current user while logging in or loading navigation. But if we have `Loadable.match` in the file, ought to make use of it to prevent this.